src/components/WwwFrame/EndOfYearAppealBanner/AppealBanner.vue

```diff
@@ -128,8 +128,17 @@ export default {
 		query: appealBannerQuery,
 		preFetch: true,
 		result({ data }) {
-			this.appealEnabled = JSON.parse(_get(data, 'general.appeal_enabled.value', false));
-			this.appealMatchEnabled = JSON.parse(_get(data, 'general.appeal_match_enabled.value', false));
+			try {
+				this.appealEnabled = JSON.parse(_get(data, 'general.appeal_enabled.value', false));
```

Comment:
@emuvente Did you see these blocks failing when working on setting this up?

Reply:
Yeah these were failing with the mock data because apollo server defaults to using `"Hello, World!"` as the value for all `String` type fields. Obviously we should make the mock data better so that doesn't happen, but it did highlight how this code can fail if the data is in an unexpected shape.